docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/dm/master-docinfo.xml

```diff
@@ -0,0 +1,10 @@
+<productname>{PRODUCT}</productname>
+<productnumber>{PRODUCT_VERSION}</productnumber>
+<subtitle>
+	For {PRODUCT} {PRODUCT_VERSION}
+
+</subtitle>
+<abstract>
+	<para>This document describes how to design a decision service using guided decision tables in {PRODUCT} {PRODUCT_VERSION}.</para>
```

Comment:
Is not about authoring pages instead of "decisions"?

Reply:
abstract changed in/dm/masterdocinfo.xml.